src/Forms/Input/MultiSelect.php

```diff
@@ -93,7 +93,7 @@ public function getID()
     public function addSortableAttribute($attribute, $values)
     {
         $this->sortableAttributes[$attribute] = $values;
-        $this->sortBySelect->fromArray(array($attribute => __("Sort by " . $attribute)));
+        $this->sortBySelect->fromArray(array($attribute => __("Sort by ") . $attribute));
```

Comment:
If we're to modify this to create a new translation string anyway, we should give a placeholder to attribute instead of just concat string:

```php
$this->sortBySelect->fromArray(array($attribute => __("Sort by {attribute}", ['attribute' => $attribute])));
```

See the `Locale::translate` function for [how it works](https://github.com/GibbonEdu/core/blob/4b7c173e03ebe65b1852f1a1fb3efe591827315c/src/Gibbon/Locale.php#L233-L283).

Reply:
I know Locale::translate. The question is that the term "Sort by " already exists and you don't need to create a new one.  Creating a new term would impact all language packs.

Reply:
That's OK then. I didn't notice the translation string "Sort by " exists. Thanks.

Reply:
I think we will need to register a task to refactor this also in the future.

Reply:
We'll probably need to mechanically create the new translation string, if possible.